scripts/upgrade-via-npm.ps1

```
@@ -18,40 +26,102 @@ param(
     [switch]$NoRestart,
     [string]$Tag = "",
     [string]$Pkg = "",
-    [switch]$Help
+    [switch]$Help,
+    [int]$Timeout = 1000,
+    [switch]$DisableBuiltin
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'DisableBuiltin' has been declared but not used.

(PSReviewUnusedParameter)